src/app/components/BookCard/styles.ts

```diff
@@ -0,0 +1,33 @@
+import {StyleSheet} from 'react-native';
+
+export default StyleSheet.create({
+  card: {
+    alignItems: 'flex-start',
+    backgroundColor: 'white',
```

Comment:
You should declare all the colors with hexadecimals and in a file of constants in `src/constants/colors`